ts_src/index.ts

```diff
@@ -207,6 +259,7 @@ export function validateMnemonic(
   try {
     mnemonicToEntropy(mnemonic, wordlist);
   } catch (e) {
+    console.log('could not validate mnemonic', e);
```

Comment:
So... I made a deliberate decision to preserve this console log when I reviewed/cherry-picked this patch in, but I don't remember why and it seems inappropriate to me now for this to be in a library.

```suggestion
```

Reply:
done here: https://github.com/MetaMask/bip39/pull/1/commits/f5dcd825314b3e9b32e28bf0eb51ae8a9d27e240

Reply:
scratch that. Done here: https://github.com/MetaMask/bip39/commit/a01d40d19791de9cee60a80d87d35b61feb6dc97